src/store.js

```diff
@@ -39,14 +40,23 @@ store.dispatch(setFilter(getTags(lessonContext, courseContext)));
 let initialMode = true;
 let initialLanguage = 'nb';
 let initialWelcomeBox = true;
+let initialPath = '/scratch/astrokatt/astrokatt';
+let initialCheckboxes = {};
 
 if (typeof localStorage !== 'undefined') {
   if (localStorage.isStudentMode) { initialMode = JSON.parse(localStorage.isStudentMode); }
   if (localStorage.language) { initialLanguage = localStorage.language; }
   if (localStorage.welcomeBox) { initialWelcomeBox = JSON.parse(localStorage.welcomeBox); }
+  for (let i in localStorage) {
+    if(i !== 'isStudentMode' && i !== 'language' && i !== 'welcomeBox') {
+      initialPath = i;
+      initialCheckboxes = JSON.parse(localStorage[i]);
```

Comment:
Når det gjelder initialPath og initialCheckboxes, så forstår jeg ikke. Burde ikke initialPath være definert som en egen greie i localStorage? Eller hva er den til? Og initialCheckboxes? Burde ikke den inneholde checkboxes for ALLE lessons?

Nå hadde vi kanskje ikke treng å laste opp alle checkboxes for alle oppgaver med én gang (det kunne jo bli veldig mange), man kunne ta det per lesson, men greit å begynne et sted, så sikkert ok for nå å bare laste alle opp i starten.

Men jeg ville kanskje brukt ÉN key per lesson, slik at alle checkboxes for ÉN lesson var lagret i en json per lesson. Og keyene for checkboxes må kunne kjennes igjen, f.eks. "checkboxes_/scratch/astrokatt/straffespark" eller noe. Dermed kunne man nøyd seg med å se på keys som startet med "checkboxes_".

Mulig jeg har misforstått hva du mente å gjøre her, og da må du gjerne forklare.